language/move-lang/tests/move_check/parser/module_use_after_func.move

```diff
@@ -0,0 +1,5 @@
+module M {
+    f() {}
+    // Use declarations must come before function declarations.
```

Comment:
This one will state for a bit (if not always)

I'm thinking of allowing use declarations at the beginning of any block `{ }`, wether it be for declaring an expression or a module